src/nvim/api/fold.c

```diff
@@ -0,0 +1,42 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "nvim/api/fold.h"
+#include "nvim/api/private/validate.h"
+#include "nvim/fold.h"
+#include "nvim/pos_defs.h"
+
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "api/fold.c.generated.h"  // IWYU pragma: keep
+#endif
+
+/// Gets the start and end lines for the closed fold on the given line. If the line is not within a
+/// closed fold, an empty table will be returned.
+///
+/// @param window   |window-ID|, or 0 for current window
+/// @param lnum   The line number to check (0-indexed)
+/// @param[out] err Error details, if any
+/// @return {start, stop}
+ArrayOf(Integer, 2) nvim__fold_info_for_line(Window window, Integer lnum, Arena *arena, Error *err)
+  FUNC_API_SINCE(12)
```

Comment:
```suggestion
  FUNC_API_SINCE(13)
```
(or even higher?)

Reply:
yeah 14, and need to bump NVIM_API_LEVEL and change `NVIM_API_PRERELEASE` to `true`:  https://github.com/neovim/neovim/blob/95ab723995f1cc1a11c62355fbac64597526d47e/CMakeLists.txt#L148 (in this PR)